SocialCareCaseViewerApi.Tests/V1/Gateways/Search/GetPersonRecordsBySearchQueryTests.cs

```diff
@@ -141,6 +141,25 @@ public void WithFirstnameInNameQueryParameterReturnsMatchingResident()
             listOfPersons.Should().ContainEquivalentOf(DatabaseContext.Persons.First(x => x.Id == person2.Id).ToResidentInformationResponse());
         }
 
+        [Test]
+        public void WildcardSearchWithFirstNameInFirstNameQueryParameterReturnsMatchingResident()
+        {
+            var person1 = DatabaseGatewayHelper.CreatePersonDatabaseEntity(firstName: "ciasom");
+            var person2 = DatabaseGatewayHelper.CreatePersonDatabaseEntity(firstName: "Ciasom");
```

Comment:
Maybe worth sticking a person with first name not matching the request, to explicitly show that it doesn't just returns everything there is, however this was already tested using `Name` in request...

Reply:
I'll be covering this in a follow up commit where I contract the API to remove the Name parameter, see the ticket for details.

Reply:
Awesome!